model/CreatorConfig.php

```diff
@@ -99,6 +121,9 @@ public function init(){
         foreach($this->infoControls as $infoControl){
             $this->prepare($infoControl);
         }
+        foreach ($this->controlEndpoints as $key => $controlEndpoint) {
+            $this->setProperty($key, \tao_helpers_Uri::getRootUrl() . $controlEndpoint);
```

Comment:
One last thing, the backend team just confirmed me the best practice for URLs is to use the helper `url` instead of concat the root url : 

```php
'itemDataUrl' => \tao_helpers_Uri::url('getItemData', 'QtiCreator', 'taoQtiItem'),
```

Could you please make this last change ?